.taskcluster.yml

```diff
@@ -177,6 +177,35 @@ tasks:
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("update_hook")}
+        scopes:
+          - hooks:modify-hook:project-relman/bugbug
+          - assume:hook-id:project-relman/bugbug
```

Comment:
We should check if it works after removing this (maybe not, but just want to be sure)

Reply:
I had to re-add it, it was failing without https://tools.taskcluster.net/groups/NJps5ECFTciS6xf0sfSEqw/tasks/fbJ6T3rfTG-oq2x9h1NtfQ/runs/0/logs/public%2Flogs%2Flive.log, maybe you dit changed something since?

Reply:
OK, so yes, let's keep it.